index.js

```diff
@@ -60,15 +60,40 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
+const header = document.querySelector("h1#header")
 function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+  if (element.target.style.color === "red") {
+    element.target.style.color = "black"
   } else {
-    element.style.color = "red"
+    element.target.style.color = "red"
   }
 }
 
+header.addEventListener("click", toggleColor)
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector(".like-button")
+likeButton.addEventListener("click", like)
 
+function like(){
+  const popularity = document.querySelector(".likes")
+  const likes = parseInt(popularity.textContent) + 1
+  popularity.textContent = `${likes} Likes`
```

Comment:
Looks good! The key here is that you have to _read_ some information from the DOM to get the current number of likes, increment it, and then _write_ it back to the DOM. Nice work!